geofront/backends/cloud.py

```diff
@@ -446,3 +449,21 @@ def save(self, master_key: PKey) -> None:
             driver.delete_key_pair(key_pair)
         driver.import_key_pair_from_string(self.key_pair_name, public_key)
         self.master_key_store.save(master_key)
+
+
+def create_cloud_master_pubkey_store(compute_provider_name: str,
+                                     storage_provider_name: str,
+                                     common_creds: Sequence[str],
+                                     keypair_name: str,
+                                     container_name: str,
+                                     object_name: str,
+                                     **common_driver_kwargs) -> CloudMasterPublicKeyStore:
+    """A shortcut CloudMasterPublicKeyStore factory for environment-variable
+    configs"""
```

Comment:
- Please append a period to it.
- As this docstring consists of more than a line, it should have a trailing empty line.  (See also other classes/functions' docstrings.)